packages/test-in-console/run.sh

```
@@ -21,9 +28,51 @@ export PATH=$METEOR_HOME:$PATH
+      ;;
+    *)
+      # Treat the first non-flag argument as the package filter.
+      if [[ -z "$PACKAGE_FILTER" ]]; then
+        PACKAGE_FILTER="$arg"
+      fi
+      ;;
+  esac
+done
+
+export URL="http://127.0.0.1:$PORT/"
+MONGO_PORT=$(($PORT + 1))
```

Shellcheck (0.11.0)

[style] $/${} is unnecessary on arithmetic variables.

(SC2004)